lib/js/index.js

```diff
@@ -216,6 +240,20 @@ class DateTimePicker extends Events {
     return this;
   }
 
+  showHourClock() {
+    this.$('.c-datepicker__clock__hours').style.display = 'block';
```

Comment:
It seems inconsistent to use `style.display` for one part of the UI and a className for the other. Could you use an `active` class for both?